src/lpf2hub.ts

```diff
@@ -20,6 +20,10 @@ export class LPF2Hub extends Hub {
         return [t[0], t[1], t.substring(2, 4), t.substring(4)].join(".");
     }
 
+    private static decodeMacAddress(v: Uint8Array) {
+        return Array.from(v).map((n) => ("00" + n.toString(16)).slice(-2)).join(":");
```

Comment:
This is simpler now with ECMASctipt 2017. I also happened to add an utility function `toHex` which does exactly ` n.toString(16).padStart(2, "0")` in my own PR.

```suggestion
        return Array.from(v).map((n) => n.toString(16).padStart(2, "0")).join(":");
```

Reply:
I will keep in mind to use your `toHex` as soon as it get merged.

Reply:
I merged @nutki's PR, feel free to use `toHex` now :)